caluma/caluma_workflow/api.py

```diff
@@ -0,0 +1,24 @@
+from . import domain_logic
+
+
+def start_case(*args, **kwargs):
```

Comment:
Please use `*args` and `**kwargs` only for optional or unknown parameters. Known and required parameters should be named (optional ones should have a default value).

In this case, `workflow` is required, `form` is known-but-optional, as is `user`. Any others shold be documented

Reply:
Fair point. However, `user` is required as well. But I won't document all `**kwargs` since that's just what will be passed to `Case.objects.create`.. Or do I missunderstand you?

Reply:
Done